src/Aspire.Hosting/Utils/VolumeNameGenerator.cs

```diff
@@ -24,6 +34,11 @@ public static string CreateVolumeName<T>(IResourceBuilder<T> builder, string suf
         return $"{safeApplicationName}-{applicationHash}-{resourceName}-{suffix}";
     }
 
+    /// <summary>
+    /// Sanitizes the application name to be used in the volume name.
+    /// </summary>
+    /// <param name="name">The application name to be sanitized.</param>
+    /// <returns>The sanitized application name.</returns>
```

Comment:
Sanitize seems it does not need to be public.

Reply:
Agree, there’s some existing things referencing it, I’ll go through those and see what needs to happen to make it private

Reply:
I updated to change the visibility of Sanitize to internal